cloudinit/config/cc_resizefs.py

```diff
@@ -106,6 +106,11 @@ def _resize_hammer2(mount_point, devpth):
     return ("hammer2", "growfs", mount_point)
 
 
+def _resize_bcachefs(mount_point, devpth):
+    """Single device resize"""
+    return ("bcachefs", "resize", devpth)
```

Comment:
Am I reading the docs wrong?
```suggestion
    return ("bcachefs", "device", "resize", devpth)
```

Reply:
heh, that's probably why the test failed